.gitignore

```diff
@@ -57,3 +57,5 @@ typings/
 # dotenv environment variables file
 .env
 
+# ormconfig file
+ormconfig.json
```

Comment:
I think this file can be added to version control, In the file, we can write like this:

```js
dbname: Environment.DB.name,    // the value from process.env.NODE_ENV or other envs
```

This file does not contain security information. Of course, you're still at work. It's just a suggestion.

Reply:
Yes it does, if you look, there's an `ormconfig.example.json` file which can be copied and edited. It's a JSON file, not a JS file. And it's bad practice to store secrets in code anyway

Reply:
All secrets related to production environment are in `.env` file, other secrets of development and debugging are in file `configs/environments`.

Is it possible for us to do so?

<br />

In `.env` file:
```
MY_SECRET: 'hi',
```

<br />

In `ormconfig.js` file:
```
import { env } from 'config/environemnts'

DB_SECRET: env.MY_SECRET,
```

Reply:
Because different databases need to be automatically linked when starting applications in multiple environments.